postfixadmin/content.md

```diff
@@ -0,0 +1,73 @@
+# What is Postfix Admin?
+
+Postfix Admin is a web based interface to configure and manage a Postfix based email server for many users. Features include support for virtual domains and aliases, quotas, and vacation/out-of-the-office messages. It requires PHP, Postfix and one of MySQL, PostgreSQL or SQLite.
+
+%%LOGO%%
+
+# How to use this image
+
+## No config.local.php / no existing setup
+
+If you do not have a config.local.php, then we fall back to look for environment variables to generate one.
+
+```console
+$ docker run -e POSTFIXADMIN_DB_TYPE=mysqli \
+           -e POSTFIXADMIN_DB_HOST=whatever \
+           -e POSTFIXADMIN_DB_USER=user \
+           -e POSTFIXADMIN_DB_PASSWORD=topsecret \
+           -e POSTFIXADMIN_DB_NAME=postfixadmin \
+           --name some-%%REPO%% \
+        %%IMAGE%%
+```
+
+`POSTFIXADMIN_DB_TYPE` can be one of :
+
+-	mysqli
+-	pgsql
+-	sqlite
+
+Note: An SQLite database is not recommend but used as a fallback if you do not have a config.local.php and do not specify the above variables. Do not forget to add a volume for the SQLite path.
+
+You can also specify a `POSTFIXADMIN_SETUP_PASSWORD` environment variable.
+
+If you'd like to be able to access the instance from the host without the container's IP, standard port mappings can be used:
+
+```console
+$ docker run -e POSTFIXADMIN_DB_TYPE=mysqli \
+           -e POSTFIXADMIN_DB_HOST=whatever \
+           -e POSTFIXADMIN_DB_USER=user \
+           -e POSTFIXADMIN_DB_PASSWORD=topsecret \
+           -e POSTFIXADMIN_DB_NAME=postfixadmin \
+           --name some-%%REPO%% \
+           -p 8080:80
+        %%IMAGE%%
+```
+
+Then, access it via `http://localhost:8080` or `http://host-ip:8080` in a browser.
+
+## Existing config.local.php
+
+```console
+$ docker run --name some-%%REPO%% -p 8080:80 %%image%%
+$ docker run --name some-%%REPO%% -v /local/path/to/config.local.php:/var/www/html/config.local.php -p 8080:80 %%IMAGE%%
+```
+
+## Image Variants
```

Comment:
This section is going to look really weird next to the section that `update.sh` is going to auto-generate with the same title.  I think you wanted to instead put these blurbs into `variant-apache.md`, etc files.